code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -712,7 +713,9 @@ GLOBAL_LIST_EMPTY(PDAs)
 	var/t = stripped_input(U, "Please enter message", name)
 	if (!t || toff)
 		return
-	if(!U.canUseTopic(src, BE_CLOSE))
+	//SKYRAT EDIT CHANGE BEGIN
```

Comment:
```suggestion
	//SKYRAT EDIT CHANGE BEGIN
	//if(!U.canUseTopic(src, BE_CLOSE)) - SKYRAT EDIT - ORIGINAL
```

Reply:
Oh, sorry, I didn't actually see that last bit at the bottom about changes, I think my eyes just kind of skipped over it. Also I got no clue, the .dme does its own thing most of the time to be honest.